### PR TITLE
Remove Tectonic Settings Components

### PR DESCRIPTION
**Pull request by alecmerdler**
### Descriptions

Removes unused components from Tectonic settings (LDAP, monitoring, cluster updates).

### CODE REVIEW

**Comment by spadgett** [comment]
We're about to bring cluster updates back for OpenShift 4.0. The APIs will change, but we might be able to reuse some of the UI. I'd like to hold off just a bit to look at it.

/hold

**Comment by spadgett** [comment]
/reopen
/hold cancel

I looked at this some, and I think it will be easier to do cluster settings greenfield. I'm good with removing it. We always have git history if we need it.

We should remove the backend components, too.

File: frontend/public/components/cluster-settings/cluster-settings.jsx [hunk 1]
```diff
@@ -1,55 +0,0 @@
...
-import {SafetyFirst} from '../safety-first';
-import {FLAGS, connectToFlags} from '../../features';
-import {TectonicChannel} from '../channel-operators/tectonic-channel';
-import {ContainerLinuxUpdates} from '../container-linux-update-operator/container-linux-updates';
-
-export const SettingsRow = ({children}) => <div className="row co-m-form-row">{children}</div>;
-export const SettingsLabel = ({children}) => <div className="col-sm-4 col-md-3"><strong>{children}</strong></div>;
-export const SettingsContent = ({children}) => <div className="col-sm-8 col-md-9">{children}</div>;
-
-export const ClusterSettingsPage = connectToFlags(FLAGS.CLUSTER_UPDATES)(
```
Comment (spadgett) [inline frontend/public/components/cluster-settings/cluster-settings.jsx]: We should remove this flag from features.ts.

Comment (alecmerdler) [reply frontend/public/components/cluster-settings/cluster-settings.jsx]: @spadgett Removed server and feature detection code.

**Comment by spadgett** [comment]
```
ERROR in /go/src/github.com/openshift/console/frontend/public/features.ts
(7,40): 'ChannelOperatorConfigModel' is declared but its value is never read.
```

**Pull request merged**